example_project/mappers/base_workflow.osw

```diff
@@ -92,6 +92,17 @@
         "add_hvac": true,
         "use_upstream_args": false
       }
+    },{
+      "measure_dir_name": "add_ems_emissions_reporting",
+      "arguments": {
+        "__SKIP__": true,
+        "future_subregion": "RMPAc",
+        "hourly_historical_subregion": "Rocky Mountains",
+        "annual_historical_subregion": "RMPA",
+        "future_year": 2020,
+        "hourly_historical_year": 2019,
```

Comment:
what historical and future years are available to choose from?

Reply:
I defined these in geojson schema